### PR TITLE
feat/US34389: Multi Sig Text Proposal

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**
**Added New Multi Sig Text Proposal UI also added new UI for `Link to Discussion`**

Screenshot 2023-08-07 at 1 31 20 PM
Screenshot 2023-08-07 at 1 31 29 PM
Screenshot 2023-08-07 at 1 31 42 PM

**Also added `Link to Discussion` for `GOV` and `NFT` DAO**

Screenshot 2023-08-07 at 1 54 22 PM

### CODE REVIEW

File: src/dex-ui/pages/dao/CreateADAO/CreateADAOPage.tsx [hunk 1]
```diff
@@ -109,12 +112,12 @@ export function CreateADAOPage() {
...
     {
       label: "Details",
-      route: `${Paths.DAOs.absolute}/create/details`,
+      route: `${Paths.DAOs.absolute}/${Paths.DAOs.Create}/${Paths.DAOs.DAODetails}`,
       validate: async () => trigger(["name", "logoUrl", "isPublic", "description"]),
     },
     {
       label: "Type",
-      route: `${Paths.DAOs.absolute}/create/type`,
+      route: `${Paths.DAOs.absolute}/${Paths.DAOs.Create}/${Paths.DAOs.type}`,
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/CreateADAO/CreateADAOPage.tsx:120]: Constants should be capitalized: `Paths.DAOs.type` => `Paths.DAOs.Type`

File: src/dex-ui/services/DexService/dao/DAOService.ts [hunk 2]
```diff
@@ -405,6 +405,11 @@ export async function fetchMultiSigDAOLogs(daoAccountId: string): Promise<ethers
           receiver: event.args.info.to,
           token: TokenId.fromString(HBARTokenId).toSolidityAddress(),
         };
+      } else if (transactionType === MultiSigProposeTransactionType.TypeSetText) {
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/services/DexService/dao/DAOService.ts:408]: Can we quickly refactor this to use switch/case statement?

**Review by damiano-melcarne-lab49 — CHANGES_REQUESTED** [review]
Can be merged after two minor comments are addressed.

**Comment by roshan-bisht-lab49** [comment]
@damiano-melcarne-lab49  I have addressed the two comments.

**Pull request merged**